packages/tasks/src/tasks/text-generation/inference.ts

```diff
@@ -24,18 +24,21 @@ export interface TextGenerationInput {
  */
 export interface TextGenerationParameters {
 	/**
-	 * Whether to use logit sampling (true) or greedy search (false).
+	 * Best of
 	 */
-	doSample?: boolean;
+	bestOf?: number;
```

Comment:
@Narsil @OlivierDehaene I'm not sure what this parameter represents

Reply:
do_sample: bool -> Should the generation be greedy or not (it cannot be set to True if any other params is set, but sometimes you do not want to specify temp, nor top_p, nor anyhing, therefore setting do_sample=True is for that).

best_of: int -> Run `n` sampling queries, return the best (in terms of total logprob). Clashes with streaming.